tutorials/date-picker-data.md

```diff
@@ -106,22 +100,19 @@ function load_blob(url, callback) {
 
 if (localeStr == "ccp") {
     load_blob("https://storage.googleapis.com/static-493776/icu4x_2023-11-03/ccp.blob", (blob) => {
-        let dateTimeFormatter = ICU4XDateTimeFormatter.create_with_lengths(
-            ICU4XDataProvider.create_from_byte_slice(blob),
+        let dateTimeFormatter = DateTimeFormatter.createYmdtWithProvider(
+            DataProvider.createFromBlob(blob),
```

Comment:
turns out we don't currently have a constructor for `DataProvider` in languages that don't support static data: https://icu4x.unicode.org/2_0/tsdoc/classes/DataProvider.html

will add this in a followup